docs-ref-services/latest/eventgrid.md

```diff
@@ -31,26 +31,27 @@ To manage your Azure Event Grid resources like Event Grid instances, topics, and
 
 Create events, authenticate, and post to topics using the below packages
 
-### v2 of @azure/eventgrid
+### v4 of @azure/eventgrid
 
-To create events, authenticate, and post to topics, you would use the `@azure/eventgrid` package.
+To create events, authenticate, and post to topics, we recommend you use the latest version of the `@azure/eventgrid` package.
 
 | NPM Package                                                        | Reference                                                                                            |   
 | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- | 
-| [@azure/eventgrid v2](https://npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid v2](https://docs.microsoft.com/javascript/api/@azure/eventgrid) | 
+| [@azure/eventgrid v4](https://www.npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid v3](https://docs.microsoft.com/javascript/api/@azure/eventgrid/) |
```

Comment:
If we follow https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/1240/files#r648776971 and make a single section for the legacy packages, we can skip talking about version numbers here.



```suggestion
| [@azure/eventgrid](https://www.npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid](https://docs.microsoft.com/javascript/api/@azure/eventgrid/) |
```

Reply:
Do you think it would be confusing that `@azure/eventgrid` would appear in both tables (the latest table and the legacy table)?  Maybe calling it @azure/eventgrid@2.0.0 in the legacy table looks better?